server/src/main/kotlin/com/kamelia/jellyfish/rest/file/Service.kt

```diff
@@ -104,6 +96,14 @@ object FileService {
             return QueryResult.forbidden("errors.files.deletion.forbidden")
         }
 
+        FileUtils.delete(file.ownerId, file.code)
         return QueryResult.ok(Files.delete(fileId)?.toRepresentationDTO())
     }
+
+    suspend fun deleteFileByCodeAsAdmin(code: String): QueryResult<FileRepresentationDTO, List<ErrorDTO>> {
```

Comment:
I think the `AsAdmin` part of the name, although relevant, is not needed here. We don't want to create extra long methods names that describe the method logic..

Reply:
I do prefer clear names for what methods do in general ; there is virtually no cost over the benefit